README.md

```diff
@@ -116,9 +116,32 @@ and
 [ember-cp-validations](https://github.com/offirgolan/ember-cp-validations)
 respectively.
 
+The buffer can be imported and used directly:
+
+```js
+import { Buffer } from 'ember-validated-form-buffer';
+
+const Validations = buildValidations({
+  name: validator('presence', true)
+});
+
+export default Ember.Controller.extend({
+  data: computed('model', function() {
+    let owner = Ember.getOwner(this);
+    return Buffer.extend(Validations).create(owner.ownerInjection(), {
+      content: this.get('model')
```

Comment:
I think it would be good to add a sample for when a custom `Buffer` might be helpful?